src/pages/library/_viewId.vue

```diff
@@ -339,27 +339,6 @@ export default Vue.extend({
 </script>
 
 <style lang="scss" scoped>
-@import '~vuetify/src/styles/styles.sass';
```

Comment:
#1478 already cleans this up a lot by using the Vuetify values to position it properly.